sympy/geometry/tests/test_line.py

```diff
@@ -220,10 +220,10 @@ def test_line_geom():
     assert Line((0, 0), (0, 1)).distance(p2) == 1
     assert Line((0, 0), (1, 0)).distance(p1) == 0
     assert Line((0, 0), (1, 0)).distance(p2) == 1
-    m = symbols('m')
+    m = symbols('m', real=True)
     l = Line((0, 5), slope=m)
     p = Point(2, 3)
-    assert l.distance(p) == 2*abs(m + 1)/sqrt(m**2 + 1)
+    assert simplify(l.distance(p) - 2*abs(m + 1)/sqrt(m**2 + 1)).is_zero
```

Comment:
Did the distance routine lose a simplification step? The expression obtained without simplification is `sqrt((3 - (3*m**2 + 2*m + 5)/(m**2 + 1))**2 + (-(-2*m + 2)/(m**2 + 1) + 2)**2)`.